### ISSUE TITLE
FlxAsset error on ios target

### ISSUE THREAD

**Issue by cutepig49** [HaxeFlixel/flixel#1549]
```
flixel: [3.3.8]
```
```
haxe Build.hxml -D HXCPP_ARMV7 -cpp build/Release-iphoneos-v7 
/usr/lib/haxe/std/neko/_std/sys/FileSystem.hx:85: characters 27-57 : assets/: No such file or directory
/usr/lib/haxelib/flixel/3,3,8/flixel/system/FlxAssets.hx:72: characters 22-57 : Called from
/usr/lib/haxelib/flixel/3,3,8/flixel/system/FlxAssets.hx:52: characters 44-106 : Called from
../../../../source/AssetPaths.hx:3: characters 2-7 : Called from
Aborted
make: *** [build-haxe-armv7] Error 1
```

**Comment by Gama11** [comment]
More details would be nice. What Lime and OpenFL versions?

What does your code look like / what's needed to reproduce it? Does this happen with a freshly generated template project?

**Comment by cutepig49** [comment]
Yes, just create a new project, and 
```
override public function create():Void
{
	super.create();	
	trace(AssetPaths.images-go-here.txt);
}
```
then
```
lime build ios
````
it happens.

The verions:
```
flixel: [3.3.8]
lime: [2.4.4]
openfl: [3.1.0]
```

**Comment by Gama11** [comment]
Pretty sure this has already been fixed in #1294. The fix hasn't made it into a haxelib release yet though.

**Comment by Gama11** [comment]
Version 3.3.9 is now on haxelib and should fix this.
